MonoGame.Framework/Graphics/Vertices/VertexBuffer.cs

```diff
@@ -54,13 +54,18 @@ internal protected override void GraphicsDeviceResetting()
 
         public void GetData<T> (int offsetInBytes, T[] data, int startIndex, int elementCount, int vertexStride) where T : struct

```

Comment:
> I'd prefer to stick with XNA's parameter names until there is a broader
> decision to move away from the XNA API.

There is a plan in the future to move beyond XNA4, but that isn't my point here.

We do try to fix mistakes in XNA when we can.  Especially mistakes that generally don't hurt compatibility.  In this case the code will work 100% the same...  it will just make more sense.